themes/bootstrap3/templates/Recommend/SideFacets/cluster-list.phtml

```diff
@@ -23,7 +23,7 @@
     'group' => $this->title,
     'moreClass' => $i >= $this->facets_before_more ? $moreClass : false,
     'url' => $this->results->getUrlQuery(),
-    'urlBase' => $this->currentPath(),
+    'urlBase' => '',
```

Comment:
It was part of an attempt to make the tab work with async loading too, since current path would be the path to the tab instead of the page. I couldn't complete it yet, but the same change will be needed for deferred side facets anyway. There should be no problem with only the query part in the url and we've been running like that for years now, but obviously there could be an egde case that's affected.

Reply:
Makes sense. I can't think of an obvious reason for this to cause problems, and your history of using it is encouraging. I'll just add a TODO reminder to run the full test suite before merging in case there are any edge issues detected there.